### ISSUE TITLE
Turtle: Add Jewelcrafting

### ISSUE THREAD

**Issue by refaim** [refaim/LibCrafts-1.0#4]

**Comment by refaim** [comment]
https://github.com/ImmortalSom/MasterTradeSkills

**Comment by manuf10** [comment]
Hey! What would be the steps to add Jewelcrafting data? I'm new in addons development. I guess one would need to create a Jewelcrafting.lua file and put static data about all the recipes there.
Would https://database.turtle-wow.org be the place to scrap that data? (It's still missing on their side tho)

**Comment by refaim** [comment]
@manuf10 

Yes, you'd need to create a file and add recipe data there. @ImmortalSom has already done some of this work in [their fork of MasterTradeSkills](https://github.com/ImmortalSom/MasterTradeSkills/blob/master/lib/LibCrafts-1.0/Professions/Jewelcrafting.lua), but it would be better to implement it here in LibCrafts so other addons can benefit from it too.

Normally we'd scrape data from the turtle database, but since Jewelcrafting recipes aren't there (and aren't planned to be added as far as I know), you can find some info in @Lexiebean's [AtlasLoot fork](https://github.com/Lexiebean/AtlasLoot/blob/main/Crafting/crafting.en.lua) or simply collect recipes manually while playing (you'll probably need to use fake spell IDs for now—I'm not sure if they can be extracted from the game client directly).

**Comment by refaim** [comment]
Added in https://github.com/refaim/LibCrafts-1.0/commit/20746bb7b75dc2c854c9469dc8ef6e053be9dad3
